src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,14 @@
      * "Can't get min value from list: method_input_list"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(x -> x.split(","))
+                .flatMap(x -> Arrays.stream(x.clone()))
```

Comment:
```suggestion
                .flatMap(x -> Arrays.stream(x.split(",")))
```
you can remove first map operation